.github/workflows/release.yml

```diff
@@ -38,31 +39,33 @@ jobs:
           echo "PROJECT_VERSION=${GITHUB_REF#refs/tags/v}" >> "$GITHUB_ENV"
           echo "Version: ${{ env.PROJECT_VERSION }}"
 
+      - name: Check git tag against toml values
+        if: ${{ matrix.job.os != 'windows-2019' }} # windows too weird
+        run: |
+          grep -q "version = \"$PROJECT_VERSION\"" Cargo.toml
+          grep -q "version = \"$PROJECT_VERSION\"" wapm.toml
+
       - name: Build release
-        run: cargo build --locked --verbose --release --target=${{ matrix.job.target }}
+        run: cargo build --locked --verbose --release --target=${{ matrix.job.target }} --no-default-features --features '${{ matrix.job.features }}'
+        env:
+          RUSTFLAGS: -C strip=symbols
```

Comment:
I didn't know this option, but I've checked this reduces binary size. Thanks!

Reply:
(Only reason I used it is that I didn't want to have to install wabt.)